quarkus/tests/integration/src/main/java/org/keycloak/it/junit5/extension/CLITestExtension.java

```diff
@@ -104,6 +106,16 @@ public void beforeEach(ExtensionContext context) throws Exception {
         }
     }
 
+    @Override
+    public void interceptTestMethod(Invocation<Void> invocation,
+            ReflectiveInvocationContext<Method> invocationContext, ExtensionContext extensionContext) throws Throwable {
+        if (dist == null) {
+            super.interceptTestMethod(invocation, invocationContext, extensionContext);
+        } else {
+            invocation.proceed();
+        }
+    }
```

Comment:
Just out of curiosity, what is the reason for this?

Reply:
There are changes to the Quarkus test extension which we are extending. It impacts running tests with `keepAlive`.